content/en/news/releases/1.3.x/announcing-1.3.6/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Announcing Istio 1.3.6
+description: Istio 1.3.5 release announcement.
+publishdate: 2019-12-10
+attribution: The Istio Team
+subtitle: Minor Update
+release: 1.3.6
+aliases:
+    - /news/announcing-1.3.6
+---
+
+This release contains fixes for the security vulnerability described in [our December 10, 2019 news post]()  as well as bug fixes to improve robustness. This release note describes what's different between Istio 1.3.5 and Istio 1.3.6.
```

Comment:
[our December 10, 2019 news post]()
maybe need url.

Reply:
Thanks. The date was announced but the article isn't up yet. Will update that when it's available.

Reply:
done

Reply:
Is the security update section awaiting an entry?

Reply:
Yep. 1.3.6 will include the issues mentioned in this announcement: 

https://discuss.istio.io/t/upcoming-security-releases-of-istio-1-2-10-1-3-6-and-1-4-1/4553